### ISSUE TITLE
Calling .replaceWithText() on a node with a jsdoc replaces the node but leaves the jsdoc

### ISSUE THREAD

**Issue by gregjacobs** [dsherret/ts-simple-ast#178]
Hey David, not sure if this should actually happen or not, but seems like it might make sense if `.replaceWithText()` removed the jsdoc as well?

```ts
import Ast from 'ts-simple-ast';

const ast = new Ast();
const sourceFile = ast.addSourceFileFromText( 'test.ts', `
	/**
	 * Some class
	 */ 
	export class Stuff {}
`);

const myClass = sourceFile.getClasses()[ 0 ];
myClass.replaceWithText( 'var a = 1;' );

console.log( sourceFile.getFullText() );
```

Currently the output of this is:

```ts
/**
 * Some class
 */ 
var a = 1;
```

Thoughts on this? Should the jsdoc comment be replaced as well?

**Comment by dsherret** [comment]
Makes sense to me. Fixed in 4.1.0.

**Comment by gregjacobs** [comment]
Cool, sounds good, thx!
